Index.html

```diff
@@ -1,269 +1,176 @@
-                    next project
-                    <i class="fa fa-light fa-arrow-right"></i>
-                </button>
-            </div>
-        </section>
-    </main>
-
-    <footer class="footer">
-        <button class="buttons footer__button">
-            <h4 class="footer__button__text">Get my resume</h4>
-            <img class="footer__button__image" src="./Contents/Resources/Icons/download_Resume.svg" alt="download my resume">
-        </button>
-
-        <ul class="socials">
-            <li href=""><img src="./Contents/Resources/Icons/ic_stackoverflow.svg" alt="go to my stackoverflow" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_github.svg" alt="go to my github" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_twitter.svg" alt="go to my twitter" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_linkedin.svg" alt="go to my linkendin" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_medium.svg" alt="go to my M" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/fingers.svg" alt="go to my M" class="social"></li>
+      </li>
+      <li>Frameworks
+        <ul id="frameworks">
+          <li>Bootstrap</li>
+          <li>Django</li>
+          <li>Rest</li>
+          <li>Celery</li>
+          <li>Postgres</li>
+        </ul>
+      </li>
+      <li>Skills
+        <ul id="skills">
+          <li>Problem solving</li>
+          <li>Agile</li>
+          <li>GitHub</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
         </ul>
-    </footer>
+      </li>
+    </ul>
+  </section>
+  <form action="https://formspree.io/f/xeqwnkjg" method="POST" id="contact-form">
+    <h1>Get Started</h1>
+    <p>I prefer to speak by my gestgures.
+    </p>
+    <fieldset>
+      <label for="customer-name">
+        Your name <span class="errorText"></span>
+        <br />
+        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
+      </label>
+    </fieldset>
+    <fieldset>
+      <label for="customer-email">
+        Your email <span class="errorText"></span>
+        <br />
+        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
+      </label>
+    </fieldset>
+    <span class="errorText"></span>
+    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
+      required></textarea>
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
+  <section id="footer">
+    <a href="https://www.linkedin.com/in/mohammad-mehdi-pazouki/" target="_blank" rel="noopener">Get my CV &#10515;</a>
+    <ul>
```

Comment:
- [ ] There are some issues in this footer. kindly consider fixing them following figma guidelines.
    - The `Get my CV` link should be aligned  to left side and the social media links to the right as per figma design.
    - There is some white space beneath your footer that doesn't exist in figma design so consider removing that white space.
your design

![y](https://user-images.githubusercontent.com/96683843/236272753-dc9b8389-63e4-4d32-bf62-7f4a4fb5ecdb.jpg)

figma design
![m](https://user-images.githubusercontent.com/96683843/236272637-c13ccdbe-0614-4de3-a0d7-4e923bd6c04b.jpg)